Plugins/GitHub3/GitHubRepo.cs

```diff
@@ -131,7 +131,10 @@ public int CreatePullRequest(string myBranch, string remoteBranch, string title,
 
         public string CloneProtocol { get; set; } = "SSH";
 
-        public string[] SupportedCloneProtocols { get; set; } = new string[] { "SSH", "HTTPS" };
+        public const string SshProtocol = "SSH";
+        public const string HttpsProtocol = "HTTPS";
+        public string[] SupportedCloneProtocols { get; set; } = { SshProtocol, HttpsProtocol };
```

Comment:
Retuning arrays is rather dangerous, let's fix it
```suggestion
        public IReadOnlyList<string> SupportedCloneProtocols { get; set; } = { SshProtocol, HttpsProtocol };
```